include/uapi/sound/sof/tokens.h

```diff
@@ -60,6 +60,9 @@
 /* PCM */
 #define SOF_TKN_PCM_DMAC_CONFIG			353
 
+/* Stream */
+#define SOF_TKN_STREAM_AVAIL_ON_LP		380
```

Comment:
When went through topology PCM definition today, I realized that we don't need introduce new token for this, I will change to reuse the existed voice_wakeup flag in the next update.

Reply:
OK, so w.r.t. the wording, active_on_lp or suspend_active are both confusing to me, it can be active on low-power(or on suspended, or in D0i3), it can be stopped (on low-power/suspend/D0i3) also? I wanted to illustrate the capability of the stream, that's why I used the wording "available", it can be used on low-power, but you can stop it at suspend also.

@plbossart what do you think?

Reply:
> Need to find a different wording.
> 
> Also you'd want to start a new category for power-related tokens, aligned on a 100 boundary.

I thought this is stream capability, that's why I put it here near the "PCM" category, and there is no 100 boundary left here. Are you proposing to put it to the end, e.g.
#define SOF_TKN_STREAM_AVAIL_ON_LP		1000

Reply:
> #define SOF_TKN_STREAM_COMPATIBLE_D0ix 1000 ?

sounds good to me, let me change.

Reply:
@plbossart as stated below, since we are aligned to set the flag direction dependent, so I want to put it into "PCM" category, like this:
```
diff --git a/tools/topology/sof/tokens.m4 b/tools/topology/sof/tokens.m4
index 93645ceb..d01fa55d 100644
--- a/tools/topology/sof/tokens.m4
+++ b/tools/topology/sof/tokens.m4
@@ -42,6 +42,8 @@ SectionVendorTokens."sof_src_tokens" {
 }
 
 SectionVendorTokens."sof_pcm_tokens" {
+       SOF_TKN_PCM_PLAYBACK_COMPATIBLE_D0I3            "350"
+       SOF_TKN_PCM_CAPTURE_COMPATIBLE_D0I3             "351"
        SOF_TKN_PCM_DMAC_CONFIG                 "353"
 }
```
@plbossart @ranj063 sounds good?